admin-portal/package.json

```diff
@@ -17,6 +17,7 @@
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
     "react-table": "^7.8.0",
+    "react-toastify": "^9.0.5",
```

Comment:
In this PR Nuwan has used `react-simple-toasts` Library.
https://github.com/LSFLK/MedicinesforLK/pull/32/files

Shall we use the same? Any suggestions?

Reply:
Oh i didn't know he has already used one. i personally think the react-toastify looks nicer 😅 thats why i went with that one.   i think we'd have to use custom styles for different types of messages (success, error, warning etc) if we are using the library he has used. but im okay with both. what do you think?